docs/advanced_topics/images/custom_image_model.md

```diff
@@ -67,3 +67,16 @@ work as before but would need to be updated in order to see any new images.
 
 .. autofunction:: get_image_model_string
 ```
+
+
+## Overiding upload location
+
+By default, all the image files will be saved to the "images" folder. This behavior is defined by `get_upload_to` function in the class `AbstractImage` and `AbstractRendition`.
+
+`get_upload_to` function accepts the filename as an argument. This function can be overridden to define custom behavior.
+
+```{note}
+The function only works when a Custom image models is being used.
```

Comment:
I think this note can be removed, especially now that this section is on the custom image model page.